Slim/Control/Queries.pm

```diff
@@ -900,6 +873,11 @@ sub albumsQuery {
 						push @artistIds, @{$contributorHash->{$role}->{'id'}};
 					}
 				}
+				# if not dealing with a work, put the main album artist at the top of the list
+				if (!$work) {
+					unshift @artists, $c->{'contributor.name'} if $c->{'contributor.name'};
```

Comment:
Shouldn't this be `contributor*s*.name`?

Reply:
Do you think there's more left than this? I could merge and fix this line on my end.

Reply:
Pushed now, with a couple of other changes as noted in my diff document.